code/notificationbuilder/src/main/java/com/adobe/marketing/mobile/notificationbuilder/internal/builders/ProductRatingNotificationBuilder.kt

```diff
@@ -116,6 +116,7 @@ internal object ProductRatingNotificationBuilder {
                 R.id.rating_confirm,
                 selectedRatingAction.link,
                 pushTemplate.ratingSelected.toString(),
+                pushTemplate.ratingActionList[pushTemplate.ratingSelected].type,
```

Comment:
Same case here in L111. A dismiss action is by setting `PushTemplateConstants.PushPayloadKeys.STICKY, "false"`. Can we remove that and instead use `setRemoteViewClickAction` and pass in the dismiss action?

Reply:
Also the confirm button here should dismiss the notification. So the action type should always be `DISMISS`, not `ratingActionList[pushTemplate.ratingSelected].type`. The redirection uri is already passed on L117

Reply:
Discussed offline, what you have here is correct. The action type needs to be the one from the rating selected